eos-updater-ctl/eos-updater-ctl

```diff
@@ -80,9 +80,15 @@ def dump_daemon_properties(proxy):
     print("{:>{}}: {}".format("State", width, UPDATER_STATES[s]))
 
     for x in proxy.get_cached_property_names():
-        if x != "State":
-            value = proxy.get_cached_property(x)
-            print("{:>{}}: {}".format(x, width, value))
+        if x == "State":  # Handled above
+            continue
+
+        value = proxy.get_cached_property(x)
+        if x in ("DownloadSize", "DownloadedBytes", "FullDownloadSize", "FullUnpackedSize",
+                 "UnpackedSize"):
+            value = GLib.format_size_full(value.get_int64(), GLib.FormatSizeFlags.LONG_FORMAT)
```

Comment:
`flake8` is [not happy in the CI though](https://ci.endlessm-sf.com/job/obs-eos-updater-pr/473/console):
```
--- command ---
09:57:22 /usr/bin/python3 -m flake8 /build/src/_build/../eos-updater-ctl/eos-updater-ctl
--- stdout ---
/build/src/_build/../eos-updater-ctl/eos-updater-ctl:87:80: E501 line too long (91 > 79 characters)
/build/src/_build/../eos-updater-ctl/eos-updater-ctl:89:80: E501 line too long (94 > 79 characters)
```

The other CI failure looks unrelated, so please merge after fixing this nitpick. :+1:

Reply:
I reformatted this code (but not the rest of the file) with `black`, and adjusted the `flake8` config accordingly.